Dashboard/scripts/rebuild_dev_env_macos.sh

```diff
@@ -57,7 +57,6 @@ function ensure_build_tools_are_installed
     brew update
```

Comment:
AFAIK, we can delete the whole file as any dev setup should be done by Docker. @stellanl ?